### PR TITLE
feat: base network [skip cypress]

### PR DESCRIPTION
**Pull request by foodaka**
## General Changes

- Adds network BASE to UI
- Based on Aave Governance https://governance.aave.com/t/arfc-aave-v3-deployment-on-base/13708
- https://app.aave.com/governance/proposal/?proposalId=299 

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [x]  The base branch is set to `main`
- [x]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,9 @@
     "test:coverage": "jest --coverage"
   },
   "dependencies": {
-    "@aave/contract-helpers": "1.18.2",
+    "@aave/contract-helpers": "1.18.3-f07c78a955fb1b3dac86a143a504b57bdd6d02a2.0",
```
Comment (foodaka) [inline package.json:35]: Needs update to Utils package

**Comment by grothem** [comment]
Link T-13734

File: src/ui-config/marketsConfig.tsx [hunk 2]
```diff
@@ -302,6 +304,32 @@ export const marketsData: {
...
   },
+  [CustomMarket.proto_base_v3]: {
+    marketTitle: 'Base',
+    v3: true,
+    chainId: ChainId.base,
+    enabledFeatures: {
+      incentives: true,
+      liquiditySwap: true,
+      collateralRepay: true,
+      debtSwitch: true,
```
Comment (grothem) [inline src/ui-config/marketsConfig.tsx:315]: `liquiditySwap`, `collateralRepay` and `debtSwitch` should be false or just removed here

File: src/ui-config/networksConfig.ts [hunk 3]
```diff
@@ -245,6 +245,24 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
...
+    baseAssetDecimals: 18,
+    explorerLink: 'https://basescan.org',
+    // usdMarket: true,
+    networkLogoPath: '/icons/networks/base.svg',
+    bridge: {
+      icon: '/icons/networks/base.svg',
+      name: 'Base Bridge',
+      url: 'https://bridge.base.org/',
+    },
+    ratesHistoryApiUrl: 'https://aave-api-v2.aave.com/data/rates-history',
```
Comment (grothem) [inline src/ui-config/networksConfig.ts:264]: This should be removed so the charts don't try to render on the reserve page

File: src/ui-config/marketsConfig.tsx (hunk 2 above)
Comment (foodaka) [reply src/ui-config/marketsConfig.tsx]: updated in 0acd991084f164af3520d257a3e790ee8fe62230

**Review by grothem — APPROVED** [review]

**Review by defispartan — APPROVED** [review]

**Pull request merged**